test/project_types/script/commands/enable_test.rb

```diff
@@ -31,6 +31,21 @@ def test_calls_application_enable
           extension_point_type: @ep_type,
           title: @script_name
         )
+
+        @context
```

Comment:
I think it might make sense to keep everything in this one test because it performs a successful `shopify enable` call, and this new message is a part of that